tests/ord-service/tests/api_test.go

```diff
@@ -139,7 +141,22 @@ func TestORDService(t *testing.T) {
 	intSystemHttpClient, err := clients.NewIntegrationSystemClient(ctx, intSystemCredentials)
 	require.NoError(t, err)
 
-	extIssuerCertHttpClient := CreateHttpClientWithCert(certCache.Get()[conf.ExternalClientCertSecretName].PrivateKey, certCache.Get()[conf.ExternalClientCertSecretName].Certificate, conf.SkipSSLValidation)
+	commonName := "anotherCommonName"
+	replacer := strings.NewReplacer(conf.TestProviderSubaccountID, subTenantID, conf.ExternalCertCommonName, commonName)
+	externalCertProviderConfig := certprovider.ExternalCertProviderConfig{
+		ExternalClientCertTestSecretName:      conf.ExternalCertProviderConfig.ExternalClientCertTestSecretName,
+		ExternalClientCertTestSecretNamespace: conf.ExternalCertProviderConfig.ExternalClientCertTestSecretNamespace,
+		CertSvcInstanceTestSecretName:         conf.CertSvcInstanceTestIntSystemSecretName,
```

Comment:
We want a certificate that is not part of the subject mapping and at the same time is issued by our compass provider cert svc instance. This means that we need to use the original secret that is used by the cert-rotation-job.

The name of the env variable is strange, I agree, thought about it and what would be the proper name. However, this was the name used in director and I was not sure what was the context about such naming. Therefore used the same for consistency. I am open for improvements here.

Reply:
Initially, I believe the CERT_SVC_INSTANCE_TEST_INTEGRATION_SYSTEM_SECRET_NAME in the chart was created to test only one flow with the subject mapping and integration system names(CN, secrets, etc.) but my guess is gradually the scope has been expanded but the reference is not adapted and left with the initial names. I saw this is used in three places - `certificates_access_test.go`(which was the initial file/flow), `formation_mapping_api_test.go`(where the intent was to use int sys mapping, however it's already adapted/change in your PR) and the final file is `technical_client_test.go` where in my opinion the name doesn't fit.

I'm okay if you decide to merge it that way, it's a minor/small change. But if you decide to change/rename it, we could rename it to `CERT_SVC_INSTANCE_SECRET_NAME` or something similar because it refers to a cert svc instance secret(locally and productively).